Wrong shot number:
[6, 7, 11]

Correct shot number:
[6]

Raw json:

{'hasError': True, 'errParameter': ['alluxio.underfs.oss.connection.ttl'], 'reason': ["The property 'alluxio.underfs.oss.connection.ttl' has the value '-2' which is not within the accepted value {1,2,3,4,5,6,7,8,9,10}."]}

{'hasError': False, 'errParameter': [], 'reason': []}

{'hasError': True, 'errParameter': ['alluxio.underfs.oss.connection.ttl'], 'reason': ["The property 'alluxio.underfs.oss.connection.ttl' has the value '-2' which is not within the accepted value {0,1,2,3,4,5,6,7,8,9,10}."]}

Final result:

There are 1 misconfiguration parameters in the input: alluxio.underfs.oss.connection.ttl
[Ciri] Reason for alluxio.underfs.oss.connection.ttl: The property 'alluxio.underfs.oss.connection.ttl' has the value '-2' which is not within the accepted value {1,2,3,4,5,6,7,8,9,10}.
